add case studies presentation (#4)
nmfs-general-modeling-tools/presentations: docs/slides_20260422_case-studies.Rmd

  - [Southern New England-Mid Atlantic Yellowtail Flounder](https://noaa-fims.github.io/case-studies/content/NEFSC-yellowtail.html) original model in ASAP, converted to FIMS by Chris Legault

  - [Gulf of Alaska Walleye Pollock](https://noaa-fims.github.io/case-studies/content/AFSC-GOA-pollock.html) original model is a bespoke TMB model, converted to FIMS by Cole Monnahan
  
  - [Pacific Hake](https://noaa-fims.github.io/case-studies/content/pacific-hake.html) original model in SS3, converted to FIMS by Kelli Johnson
 
- Lessons learned (back to this presentation)

- Next steps

---

# Background on case studies

.left-68[
- Initial goal developed in 2022 was to have at least one case study for each NOAA Fisheries Science Center
- Case studies repository created in December 2023, initially with Southern New England-Mid Atlantic Yellowtail Flounder thanks to Chris Legault
- Multiple goals for the case studies, including:
  - testing FIMS features and identifying bugs
  - providing example models for users to learn from and build upon
  - comparing results between FIMS and existing models to build confidence in FIMS results

>   - prioritizing features to add to FIMS in the future based on differences between FIMS and existing models

Reviewer: ```suggestion
  - prioritizing features to add to FIMS based on differences between FIMS and existing models
```

The first case study (Yellowtail Flounder in ASAP &#x2192; FIMS) revealed several issues (all quickly resolved):
- additional recruitment parameter was being estimated
- had to fill missing survey data because NA was not allowed
- sex ratio was not included in spawning biomass calculations
  - fixed sex ratio = 0.5 now correctly applied, and flexibility to include sex structure is a priority for 2026 (see https://github.com/ioos/gsoc/issues/114)

Real world applications are messier than simulated data in terms of data availability and complexity and thus provide a valuable testing ground for FIMS development and debugging.

---

# Results of three case studies

switch to case studies website: https://noaa-fims.github.io/case-studies/

---

# Lessons learned

- When comparing models with features not yet available in FIMS, it can be helpful to revise the original model to be closer to the available options within FIMS 
  - isolates unknown differences in implementation from known differences in model structure
  - helps identify which features are most important to add to FIMS in the future to bring the FIMS case study closer to the original model

> - Keeping the case studies up to date with the changes in FIMS is a challenge, but also valuable

Reviewer: ```suggestion
- Keeping the case studies up to date with the changes in FIMS is challenging but valuable
```

  - provides a reminder for developers of needed changes to the user interface
  - provides example models for users to learn from and build upon
  - helps identify and fix bugs in FIMS

---

# Next steps

- More case studies are currently in development:

  - Pacific Sardine (SWFSC, SS3, Alex Jensen and Peter Kuriyama)

  - Opakapaka (PIFSC, SS3, Meg Oshima)

  - Petrale Sole (NWFSC, SS3, Ian Taylor)

  - South Atlantic Scamp (SEFSC, BAM, formerly led by Kyle Shertzer)

- Add additional case studies

  - a WHAM model (thanks to Emily Liljestrand)

  - an AMAK model (thanks to Jim Ianelli)

---

# Next steps (continued)

.left-68[

- Generalize code used to convert data for case studies into conversion functions that can be used by others

>   - e.g. `ss3_data_to_fims()` converter now available in the {r4ss} package

Reviewer: ```suggestion
  - e.g., `ss3_data_to_fims()` converter now available in the {r4ss} package
```

>   - standardize language across models (e.g. more alignment between column names between FIMS and RCEATTLE to ease the conversion process)

Reviewer: ```suggestion
  - standardize language across models (e.g., more alignment between column names between FIMS and RCEATTLE to ease the conversion process)
```
